xmtp_api_d14n/src/protocol/resolve/network_backoff.rs

```diff
@@ -19,30 +18,34 @@ use xmtp_proto::{
 };
 
 /// try resolve d14n dependencies based on a backoff strategy
-#[derive(Clone, Debug, Builder)]
-#[builder(setter(strip_option), build_fn(error = "ResolutionError"))]
```

Comment:
this builder was a bit annoying b/c it required more `Clone` in trait args so i got rid of it in favor of a standalone function. Using default exponential backoff for now anyway, so only one argument